import.py

```diff
@@ -80,7 +87,12 @@ def parse_username(username: str):
 for person in responses:
     person_exists = False
     for worker in existing_workers:
-        if person["name"].lower() == worker["name"].lower():
+
+        # prevent duplicate entries with same name or app username
+        if person["name"].lower() == worker["name"].lower() \
+        or person["paypal"].lower() == worker["paypal"].lower() \
+        or person["venmo"].lower() == worker["venmo"].lower() \
+        or person["cashapp"].lower() == worker["cashapp"].lower():
```

Comment:
Most often these will be blank, so it could trigger a false positive if "" == ""

Reply:
Good catch, we can check that the person contains a non empty value first.